hbase/src/main/java/org/apache/metamodel/hbase/HBaseFamilyMap.java

```diff
@@ -106,9 +105,9 @@ public String toString() {
             if (sb.length() > 1) {
                 sb.append(',');
             }
-            sb.append(Arrays.toString(entry.getKey()));
+            sb.append(new String(entry.getKey()));
             sb.append('=');
-            sb.append(Arrays.toString(entry.getValue()));
+            sb.append(new String(entry.getValue()));
```

Comment:
Isn't this problematic, since the byte-array may not be a string, and it may not even be decodeable as a string, so this could throw.

Reply:
The problem however is that the `Arrays.toString` variant returns the key/value combination of `hey` and `yo` as `[104, 101, 121]` and `[121, 111]`, which you can hardly call a String representation. where the `new String` will typically return `hey` and `yo` in that case. In case some unexpected character encoding is used it may display some shady characters, but it should still work.

Reply:
I believe that there are byte combinations that are simply invalid. So I suspect it could throw exceptions in certain cases. I'm not an expert on this however. Anyone else who has certainty?

Reply:
The behavior of this String constructor when the given bytes are not valid in the default charset is unspecified. There is another String constructor which also takes a Charset as an arguments, that method always replaces malformed-input and unmappable-character sequences with the charset's default replacement string. So maybe we should use that instead.

Reply:
The Apache Hadoop Hbase library has a utility method for this `org.apache.hadoop.hbase.util.Bytes#toString(byte[])`, so I changed it to use that instead.